webcompat/templates/home-page/form.html

```diff
@@ -15,9 +15,25 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
       <div class="r-Grid r-Grid--withGutter">
         <div class="r-Grid-cell r-minM--1of2">
           <div class="wc-Form-group js-Form-group">
+            <div class="wc-Form-information">
+              {{ form.url.label(class_='wc-Form-label') }}
+              {% if form.url.errors %}
+                {% for error in form.url.errors %}
+                  <span class="wc-Form-helpMessage">{{ error }}</span>
+                {% endfor %}
+              {% endif %}
+            </div>
+            <span class="wc-Form-validation">
+            {{ form.url(class_='wc-Form-item required',
+               placeholder='e.g. https://example.com',
+               required=True, type='url', pattern='') }}
+            </span>
+          </div>
+
+          <div class="wc-Form-group js-Form-group">
             <fieldset class="{% if form.problem_category.errors %}is-error{% endif %}">
               <div class="wc-Form-information">
-                <label class="wc-Form-label">What seems to be the trouble? <span class="wc-Form-required">*</span></label>
+                <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label>
```

Comment:
@karlcow Mon I will make this change (for both radio button field sets). my reluctance is that the old way seemed to be working ok. I kind of assume when a developer does something weird or breaks the pattern, it is a workaround for some unwanted behavior. but I am only guessing at best what was the original developer's intent.

please let me know afterward if you see issues from the backend side though. thanks